net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checks/design/checkstyle-metadata.xml

```diff
@@ -4,18 +4,6 @@
 "http://eclipse-cs.sourceforge.net/dtds/checkstyle-metadata_1_1.dtd">
 <checkstyle-metadata>
     <rule-group-metadata name="%Design.group" priority="1000">
-
-        <rule-metadata name="%DesignForExtension.name" internal-name="DesignForExtension" parent="TreeWalker">
-            <alternative-name internal-name="com.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck" />
-            <description>%DesignForExtension.desc</description>
-            <property-metadata name="ignoredAnnotations" datatype="String"
-                default-value="Test,Before,After,BeforeClass,AfterClass">
-                <description>%DesignForExtension.ignoredAnnotations</description>
-            </property-metadata>
-            <quickfix classname="net.sf.eclipsecs.ui.quickfixes.design.DesignForExtensionQuickfix" />
```

Comment:
We also lost quick fixes during the metadata project I think.

Reply:
Quick fixed is your plugin special metadata, you should keep it plugin override/extension of metadata

Reply:
This feature doesn't work since the metadata project last year. All this custom metadata is ignored by the plugin. I don't have time to restore this feature right now.

Reply:
Please disable it, one day it will be restored